Tests are failing in local maven build

Tests in spring-boot-admin-server-ui are failing

```
[ERROR] Failures: 
[ERROR]   AdminUiReactiveApplicationTest>AbstractAdminUiApplicationTest.should_contain_only_one_language:129 Status expected:<200 OK> but was:<500 INTERNAL_SERVER_ERROR>

> GET http://localhost:60963/sba-settings.js
> WebTestClient-Request-Id: [1]
> Accept: [*/*]

No content

< 500 INTERNAL_SERVER_ERROR Internal Server Error
< Referrer-Policy: [no-referrer]
< X-XSS-Protection: [1 ; mode=block]
< Cache-Control: [no-cache, no-store, max-age=0, must-revalidate]
< Pragma: [no-cache]
< Expires: [0]
< X-Content-Type-Options: [nosniff]
< X-Frame-Options: [DENY]
< Content-Type: [application/json]
< Content-Length: [151]
< Date: [Mon, 07 Dec 2020 20:57:04 GMT]
< Connection: [close]

{"timestamp":"2020-12-07T20:57:04.573+00:00","path":"/sba-settings.js","status":500,"error":"Internal Server Error","message":"","requestId":"9ae9c3d"}

[ERROR]   AdminUiReactiveApplicationTest>AbstractAdminUiApplicationTest.should_return_404_for_unknown_path:94 Status expected:<404 NOT_FOUND> but was:<500 INTERNAL_SERVER_ERROR>

> GET http://localhost:60946/unknown
> WebTestClient-Request-Id: [1]
> Accept: [text/html]

No content

< 500 INTERNAL_SERVER_ERROR Internal Server Error
< Cache-Control: [no-cache, no-store, max-age=0, must-revalidate]
< Pragma: [no-cache]
< Expires: [0]
< X-Content-Type-Options: [nosniff]
< X-Frame-Options: [DENY]
< X-XSS-Protection: [1 ; mode=block]
< Referrer-Policy: [no-referrer]
< Content-Length: [0]
< Date: [Mon, 07 Dec 2020 20:55:55 GMT]
< Connection: [close]

0 bytes of content (unknown content-type).

[ERROR]   AdminUiReactiveApplicationTest>AbstractAdminUiApplicationTest.should_return_index:41 Status expected:<200 OK> but was:<500 INTERNAL_SERVER_ERROR>

> GET http://localhost:60958/
> WebTestClient-Request-Id: [1]
> Accept: [text/html, */*]

No content

< 500 INTERNAL_SERVER_ERROR Internal Server Error
< Cache-Control: [no-cache, no-store, max-age=0, must-revalidate]
< Pragma: [no-cache]
< Expires: [0]
< X-Content-Type-Options: [nosniff]
< X-Frame-Options: [DENY]
< X-XSS-Protection: [1 ; mode=block]
< Referrer-Policy: [no-referrer]
< Content-Length: [0]
< Date: [Mon, 07 Dec 2020 20:56:47 GMT]
< Connection: [close]

0 bytes of content (unknown content-type).

[ERROR]   AdminUiReactiveApplicationTest>AbstractAdminUiApplicationTest.should_return_index_for_ui_path:80 Status expected:<200 OK> but was:<500 INTERNAL_SERVER_ERROR>

> GET http://localhost:60943/applications
> WebTestClient-Request-Id: [1]
> Accept: [text/html]

No content

< 500 INTERNAL_SERVER_ERROR Internal Server Error
< Cache-Control: [no-cache, no-store, max-age=0, must-revalidate]
< Pragma: [no-cache]
< Expires: [0]
< X-Content-Type-Options: [nosniff]
< X-Frame-Options: [DENY]
< X-XSS-Protection: [1 ; mode=block]
< Referrer-Policy: [no-referrer]
< Content-Length: [0]
< Date: [Mon, 07 Dec 2020 20:55:37 GMT]
< Connection: [close]

0 bytes of content (unknown content-type).

```

--- a/spring-boot-admin-server-ui/src/test/java/de/codecentric/boot/admin/server/ui/AdminUiReactiveApplicationTest.java
+++ b/spring-boot-admin-server-ui/src/test/java/de/codecentric/boot/admin/server/ui/AdminUiReactiveApplicationTest.java
@@ -16,7 +16,8 @@
 
 package de.codecentric.boot.admin.server.ui;
 
-import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.springframework.boot.SpringBootConfiguration;
 import org.springframework.boot.WebApplicationType;
@@ -31,21 +32,28 @@ import de.codecentric.boot.admin.server.config.EnableAdminServer;
 
 public class AdminUiReactiveApplicationTest extends AbstractAdminUiApplicationTest {
 
-	private ConfigurableApplicationContext instance;
+	private static ConfigurableApplicationContext instance;
 
-	@BeforeEach
-	public void setUp() {
-		this.instance = new SpringApplicationBuilder().sources(TestAdminApplication.class)
-				.web(WebApplicationType.REACTIVE).run("--server.port=0",
-						"--spring.boot.admin.ui.extension-resource-locations=classpath:/META-INF/test-extensions/",
-						"--spring.boot.admin.ui.available-languages=de");
+	private static Integer port;
 
-		super.setUp(this.instance.getEnvironment().getProperty("local.server.port", Integer.class, 0));
+	@BeforeAll
+	public static void setUp() {
+		instance = new SpringApplicationBuilder().sources(TestAdminApplication.class).web(WebApplicationType.REACTIVE)
+			.run("--server.port=0",
+				"--spring.boot.admin.ui.extension-resource-locations=classpath:/META-INF/test-extensions/",
+				"--spring.boot.admin.ui.available-languages=de");
+
+		port = instance.getEnvironment().getProperty("local.server.port", Integer.class, 0);
 	}
 
-	@AfterEach
-	public void shutdown() {
-		this.instance.close();
+	@AfterAll
+	public static void shutdown() {
+		instance.close();
+	}
+
+	@BeforeEach
+	public void setupEach() {
+		super.setUp(port);
 	}
 
 	@EnableAdminServer

--- a/spring-boot-admin-server-ui/src/test/java/de/codecentric/boot/admin/server/ui/AdminUiReactiveApplicationTest.java
+++ b/spring-boot-admin-server-ui/src/test/java/de/codecentric/boot/admin/server/ui/AdminUiReactiveApplicationTest.java
@@ -39,9 +39,9 @@ public class AdminUiReactiveApplicationTest extends AbstractAdminUiApplicationTe
 	@BeforeAll
 	public static void setUp() {
 		instance = new SpringApplicationBuilder().sources(TestAdminApplication.class).web(WebApplicationType.REACTIVE)
-			.run("--server.port=0",
-				"--spring.boot.admin.ui.extension-resource-locations=classpath:/META-INF/test-extensions/",
-				"--spring.boot.admin.ui.available-languages=de");
+				.run("--server.port=0",
+						"--spring.boot.admin.ui.extension-resource-locations=classpath:/META-INF/test-extensions/",
+						"--spring.boot.admin.ui.available-languages=de");
 
 		port = instance.getEnvironment().getProperty("local.server.port", Integer.class, 0);
 	}

